lib/ExportsInfo.js

```diff
@@ -678,14 +684,17 @@ class ExportsInfo {
 	 * @returns {string | string[] | false} the used name
 	 */
 	getUsedName(name, runtime) {
+		const isGlobalUsedExports =
```

Comment:
You just prevent any optimizations, here is a problem with runtime chunks and so we should fix it in `FlagDependencyUsagePlugin.js`, just disable this plugin and all works fine

Reply:
should I make FlagDependencyUsagePlugin process all modules even when config 'usedExports: global'，  or try other way to make all modules's exportsInfo tagged something like 'used', which make it be passed in function  `getUsedName`

Reply:
![GooglePasskeys_One_Pager_Digital (1).pdf](https://github.com/user-attachments/files/16385534/GooglePasskeys_One_Pager_Digital.1.pdf)

Reply:
I'm thinking about add a param for `processModule`, name `` when this 

> I think something wrong with runtimes, because worker always has own runtime

basically, it's due to the `undefined` value of `exportInfo._usedInRuntime`, and this value is only changed by `FlagDependencyUsagePlugin`,  but `this.global` make the plugin pass the deeper modules, so they won't have valid value of  `_usedInRuntime`.

I'm still thinking about how to resolve it, first resolution is to ignore this error, or make the plugin flag deeper modules when project is even configured with usedExport: 'global'

Is there any better resolution?